src/index.js

```diff
@@ -1,5 +1,6 @@
 import angular from 'angular';
 import ngTranslate from 'angular-translate';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
```

Comment:
```suggestion
import '@ovh-ux/translate-async-loader';
```

In that way you are able to inject the module with the `translate-async-loader` string.